.github/Makefile

```diff
@@ -67,20 +71,28 @@ create-branch-major-minor-release:
 prepare-next-release:
 	git checkout -b $(BACKPORT_BRANCH_NAME)
 	$(MAKE) update-labels
+	$(MAKE) git-diff
 	if [ ! -z "$$(git status -s)" ]; then \
-    	git status -s; \
-    	git add --all; \
-    	git commit -a -m "[Release] add-backport-next"; \
+		git status -s; \
+		git add --all; \
+		git commit -a -m "[Release] Add backport-$(PROJECT_MAJOR_VERSION).$(PROJECT_MINOR_VERSION)"; \
 	fi
 
 ## @help:create-prs-next-release:Create the PRs with the next release cycle.
 .PHONY: create-prs-next-release
 create-prs-next-release:
 	git checkout $(BACKPORT_BRANCH_NAME)
-	git push --set-upstream origin add-backport-next-$(NEXT_PROJECT_MINOR_VERSION)
+	git push --set-upstream origin $(BACKPORT_BRANCH_NAME)
 	gh pr create \
-		--title "backport: Add $(PROJECT_MAJOR_VERSION).$(PROJECT_MINOR_VERSION) branch" \
+		--title "backport: Add backport-$(PROJECT_MAJOR_VERSION).$(PROJECT_MINOR_VERSION) label" \
```

Comment:
More details about what's happening here, since it's just adding the backport label in the mergify file.